2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Configure stats pid to 89502
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Loading settings from /Users/ryanmarr/.config/wandb/settings
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Loading settings from /Users/ryanmarr/Documents/CognArch/GflowsForSimulation/wandb/settings
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': '/Users/ryanmarr/Documents/CognArch/GflowsForSimulation/train.py', 'program': '/Users/ryanmarr/Documents/CognArch/GflowsForSimulation/train.py'}
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_init.py:_log_setup():524] Logging user logs to /Users/ryanmarr/Documents/CognArch/GflowsForSimulation/wandb/run-20231211_162006-ey7efjpt/logs/debug.log
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/ryanmarr/Documents/CognArch/GflowsForSimulation/wandb/run-20231211_162006-ey7efjpt/logs/debug-internal.log
2023-12-11 16:20:06,826 INFO    MainThread:89502 [wandb_init.py:init():564] calling init triggers
2023-12-11 16:20:06,827 INFO    MainThread:89502 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'learning_rate': 100000.0, 'epochs': 1000}
2023-12-11 16:20:06,827 INFO    MainThread:89502 [wandb_init.py:init():614] starting backend
2023-12-11 16:20:06,827 INFO    MainThread:89502 [wandb_init.py:init():618] setting up manager
2023-12-11 16:20:06,832 INFO    MainThread:89502 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-12-11 16:20:06,834 INFO    MainThread:89502 [wandb_init.py:init():624] backend started and connected
2023-12-11 16:20:06,840 INFO    MainThread:89502 [wandb_init.py:init():716] updated telemetry
2023-12-11 16:20:06,860 INFO    MainThread:89502 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2023-12-11 16:21:37,216 ERROR   MainThread:89502 [wandb_init.py:init():775] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-12-11 16:21:39,239 ERROR   MainThread:89502 [wandb_init.py:init():1188] Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/Users/ryanmarr/Documents/CognArch/.cogn_arch/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    run = wi.init()
  File "/Users/ryanmarr/Documents/CognArch/.cogn_arch/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 781, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-12-11 16:22:10,085 WARNING MsgRouterThr:89502 [router.py:message_loop():77] message_loop has been closed
